### ISSUE TITLE
Wrong display when we have multiple app that not support all flavor

### ISSUE THREAD

**Issue by yumiki** [velgun/AndroidBuildVariantPlugin#1]
Firstly, thanks for the plugin; it works well overall.

However, I encountered an issue when there is multiple apps in the same project that don't implement the same product flavor.

Do you think this is something that can be improved?

**Comment by velgun** [comment]
Could you provide more details about issue

**Comment by yumiki** [comment]
Yeah sure,

For example, I have one project with multiple product flavors only for the TV app and none for the mobile app. Since the TV app is the first module, I can see <product-name>Debug or <product-name>Release, which is good. However, it can be confusing when I change the run configuration to the mobile app, as I still see <product-name>Debug or <product-name>Release, which doesn't make sense for the mobile app.

I have reproduced a similar architecture in a minimal project that I can share with you if you want. Here is a screenshot of what happens: 
<img width="1440" alt="Capture d’écran 2024-04-06 à 11 40 27" src="https://github.com/velgun/AndroidBuildVariantPlugin/assets/12427684/c7a11af1-b873-480b-aaea-2d3ca5044190">
The difference is that the first module is the mobile app so only debug and release is displayed. 
But since the active run configuration is tv I expect firetvDebug to be displayed instead.

**Comment by velgun** [comment]
Hello, I've resolved the issue by updating the plugin to correctly display the product flavor for the active run configuration. Now, when switching between different modules with different product flavors, the plugin shows the appropriate flavor for each module. I've tested this in a minimal project and it's working as expected. Feel free to reach out if there are any further questions or concerns. Closing the ticket now. Thank you!
